api/v2/controllers/adminController.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable padded-blocks */
+/* eslint-disable radix */
+import articles from '../models/article';
+
+class AdminAccess {
+  // delete specific flagged article
```

Comment:
There's a better way to comment on methods using Jsdocs format. For instance, you can do rewrite this comment as /**
            * @description delete specific flagged article
            * @param {Object} req
            * @param {Object} res
            * @returns {Object} Returns the response
            / ***

*Note:* Do this on every method where you have comments.

Reply:
thanks for the recomendation point noted